.github/workflows/updatecli.yml

```diff
@@ -0,0 +1,42 @@
+name: "Updatecli: Dependency Management"
+
+on:
+  schedule:
+    # Run every hour
+    - cron: '0 * * * *'
```

Comment:
I guess running this once or twice a day is sufficient, and ad-hoc if needed (manually or triggered by another event)

Reply:
Running every hour is easier to propagate the updates when they need to be done in order, like the rancher/shell in different projects. Since the GH Actions if free in public repos, I would leave every hour. It doesn't hurt.

Reply:
Just a suggestion that could reduce the occurrence of the pipeline for pipelines that are time sensitive.

On Epinio we use a custom dispatch event when an Epinio releases is finished to notify both the doc repository and the helm chart repository. 

On the doc repository, the goal is to trigger Updatecli to retrieve the latest Epinio version to create a PR with the new documentation website such as https://github.com/epinio/docs/pull/193
On the helm chart repository, the goal is to trigger Updatecli to retrieve the latest Epinio version to create a new Helm chart version with the new docker image (if available from a registry ) https://github.com/epinio/helm-charts/pull/302

Here is an example for the dispatch event
* https://github.com/epinio/epinio/blob/main/.github/workflows/release.yml#L61L69
* https://github.com/epinio/docs/blob/main/.github/workflows/updatecli.yml#L10

On all cases we want human validation, so it's part of the release process to go on each repository and validate the PRs

Reply:
Issue to track this improvement - https://github.com/rancherlabs/updatecli-automation/issues/3.